benchmark_runner/main/environment_variables.py

```diff
@@ -190,6 +191,11 @@ def __init__(self):
         self._environment_variables_dict['clusterbuster_workload'] = EnvironmentVariables.get_env('CLUSTERBUSTER_WORKLOAD', '')
         self._environment_variables_dict['clusterbuster_uuid'] = EnvironmentVariables.get_env('CLUSTERBUSTER_UUID', '')
 
+        # KRKN HUB data: Chaos testing
+        # For more details, see the documentation: https://github.com/krkn-chaos/krkn-hub?tab=readme-ov-file#supported-chaos-scenarios.
+        self._environment_variables_dict['krknhub_workload'] = EnvironmentVariables.get_env('KRKNHUB_WORKLOAD', '')
+        self._environment_variables_dict['krknhub_environment_variables'] = EnvironmentVariables.get_env('KRKNHUB_ENVIRONMENT_VARIABLES', '')
```

Comment:
What is this variable for and what does the value look like?

Reply:
See the krknhub_environment_variables in Description

Reply:
I don't see mention in the PR description (and there should be a description in the code, not just in the PR).

Reply:
See it up:
'krknhub_environment_variable' e.g.
export CLOUD_TYPE=
export BMC_USER=
export BMC_PASSWORD=
export BMC_ADDR=
export NODE_NAME=
export ENABLE_ES=
export ES_SERVER=
export ES_PORT=
export ES_METRICS_INDEX=
export ES_COLLECT_METRICS=

The environment variable is changes between the test scenarios,
I have added the document link:
        # For more details, see the documentation: https://github.com/krkn-chaos/krkn-hub?tab=readme-ov-file#supported-chaos-scenarios.

Reply:
Right, but what does the exact value of KRKNHUB_ENVIRONMENT_VARIABLES look like?  Environment variables are single strings, not a list of strings; if you want a list, it has to be encoded (typically colon separated, in the case of PATH and similar).

Reply:
Good point: Added
`# e.g. "export CLOUD_TYPE='test'; export BMC_USER='user'"`